package.json

```
@@ -66,7 +66,7 @@
     "typescript": "^3.7.4"
   },
   "engines": {
-    "node": "8.* || >= 10.*"
+    "node": ">= 10.*"
```

```suggestion
    "node": "10.* || >= 12.*"
```

Node.js 11 is EOL too 😉